packages/py-api/py_api/controllers/uploader_controller.py

```diff
@@ -69,3 +75,18 @@ def dump_objects(cls) -> Dict[str, Any]:
             )
 
         return {'objects': objects}
+
+    @classmethod
+    def delete_object_by_filename(cls, filename: str) -> Dict[str, Any]:
+        # Check whether the image exists
+        try:
+            cls._s3_client.head_object(
+                Bucket=cls._AWS_BUCKET_NAME, Key=filename,
+            )
+        except cls._s3_client.exceptions.ClientError as e:
+            if e.response['Error']['Code'] == '404':
+                return {"error": f"Object '{filename}' not found in the storage."}
+
```

Comment:
The goal was to return the status code to the end user. Nevertheless, this addition is not redundant.